string package_colour
bool task_done
